docs/secure-connections/sauce-connect/installation.md

```diff
@@ -70,6 +66,19 @@ tar xvf ./sc-4.7.1-linux.tar.gz
 export PATH="$HOME/sc-4.7.1-linux/bin:$PATH"
 ```
 
+
+## Using Docker Containers to Install/Run Sauce Connect
+
+You can use the Sauce Connect Proxy Docker image, maintained by the Sauce Labs [Open Source Program Office](https://opensource.saucelabs.com/), to build a Docker container. This means is an alternative to the above Download/Installation steps.
```

Comment:
build a Docker container -> run a Docker container

This means is -> This means *it* is